src/modules/pdf-notifications/view-helpers.js

```diff
@@ -2,7 +2,7 @@ const moment = require('moment');
 const { chunk } = require('lodash');
 const naldRegions = require('./data/nald-regions');
 const naldAreas = require('./data/nald-areas');
-const naldDates = require('../../lib/nald-dates');
+const naldDates = require('@envage/water-abstraction-helpers/src/nald/dates');
```

Comment:
Is it OK to be requiring things from the /src directory or is it best to go via an object path?